TikTok ban and alternatives for Social Media.md

```diff
@@ -0,0 +1,66 @@
+| Content | This Article will talk about The possible Tiktok ban in the U.S and highlight the problems of privacy and data misuse in the current Social Media ecosystem. The talk will be directed to highlighting alternative to the traditional Social media platforms, specially Bison Relay |
+|---|---|
+| Title               | TikTok on the clock: Th possible ban, privacy concerns, and the Potential of Decentralized Platforms |
```

Comment:
Title idea "The current state of social media". 

I'm not a fan of the word "decentralised". Alternatives could be "Peer to peer", "Censorship resistant"

e.g. The current state of social media and the Potential for peer to peer alternatives